services/storage/src/simcore_service_storage/db_file_meta_data.py

```diff
@@ -71,9 +71,11 @@ async def list_filter_with_partial_file_id(
     project_ids: list[ProjectID],
     file_id_prefix: str | None,
     partial_file_id: str | None,
+    only_files: bool,
```

Comment:
this is confusing:
- only_files = True, returns only files that is ok
- but only_files = False, I would expect to receive then files AND folders, i.e. not *only the files*, which is not what is happening here as this will actually return only folders.
--> you can:
- use an enum, that is probably the clearest way such as FileType: FILE, FOLDER, SYMLINK, ALL,
- or have another kw such as only_folders

But like this is wrong I think